frontend/src/routes/wallet/keypair/home.tsx

```diff
@@ -13,21 +13,23 @@ export function KeyPairHome() {
   }
 
   return (
-    <div style={{ padding: 20 }} data-testid='keypair-home'>
-      <Header style={{ marginTop: 0 }}>Details</Header>
-      <KeyValueTable
-        rows={[
-          { key: 'Name', value: keypair.name, dataTestId: 'keypair-name' },
-          {
-            key: 'Public key',
-            value: <BreakText>{keypair.publicKey}</BreakText>,
-            dataTestId: 'public-key'
-          }
-        ]}
-      />
-      <Header>Assets</Header>
-      <AccountsTable publicKey={keypair.publicKey} />
-    </div>
+    <>
```

Comment:
chore (blocking): Unnecessary fragment

Reply:
totally didn't remember adding this. removed!